src/tree/render.js

```diff
@@ -102,12 +102,22 @@ export function createContainer({
   expandedState,
   setExpandedState,
   viewState,
-}) {
+  setContainerData,
+}) => {
   element.innerHTML = '';
   element.className = 'sn-org-chart';
   let positioning = position('ttb', element, {});
   const { width, height } = element.getBoundingClientRect();
 
+  const homeButton = select(element)
+    .append('button')
+    .attr('class', 'sn-org-homebutton disabled')
+    .on('click', () => {
+      createContainer({ element, dataTree, selectionObj, wrapperState, setInitialZoom, setTransform, expandedState, setExpandedState, viewState, setContainerData });
+    })
+    .html('<span class="lui-fade-button__icon  lui-icon  lui-icon--large  lui-icon--home"></span>')
```

Comment:
I know I put this in here as well, but we might want to place the svg home icon here so it also works outside Sense

Reply:
yeah, it's not very pretty outside of sense